src/Xamarin.Android.Build.Tasks/Tasks/ResolveAndroidTooling.cs

```diff
@@ -192,8 +193,7 @@ public override bool RunTask ()
 
 		protected virtual bool Validate ()
 		{
-			AndroidApiLevel = GetMaxStableApiLevel ().ToString ();
-			return true;
+			return !string.IsNullOrEmpty (AndroidApiLevel);
```

Comment:
This PR might be related, I made similar changes: https://github.com/xamarin/xamarin-android/pull/4960

Reply:
Lol I should have looked through open PRs...

I think we can try to get the android-sdk-installer changes through first via this PR (without these test changes), and move the few test changes to #4960? I think some aspect of the new asserts should be carried over either way, to ensure that we're using the newly installed tools.

Reply:
Whichever one is merged first would be fine, I changed my PR to look more like this one. I had to ignore the test on Windows + `dotnet` because `platform-30`  doesn't actually install.